Packs/VaronisSaaS/Integrations/VaronisSaaS/VaronisSaaS.yml

```diff
@@ -0,0 +1,548 @@
+category: Analytics & SIEM
+commonfields:
+  id: VaronisSaaS
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+  defaultvalue: Varonis SaaS Incident
+- display: The FQDN/IP the integration should connect to
+  hidden: false
+  name: url
+  required: true
+  type: 0
+- display: X-API-Key
+  hidden: false
+  name: apiKey
+  required: true
+  type: 4
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  hidden: false
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: '1000'
+  display: Maximum number of incidents per fetch
+  additionalinfo:
+  name: max_fetch
+  type: 0
+  required: false
```

Comment:
You do not take this parmeter into consideration in the fetch_command. Please limit the number of reasults to the max_fetch

Reply:
Done